events/README.md

```diff
@@ -0,0 +1,36 @@
+# Events
+
+```
+{
+  'name': string,
+  'description': string,
+  'room_id':string,
+  'template_id': string,
+  'enabled': boolean,
+  'lock': boolean,
+  'region': <'in' | 'us' | 'eu' | 'auto'>,
+  'peers': [
+    "<peer_id>",
```

Comment:
For the one session there will be finite no. of users so we can use whole object here instead of referencing.
Also for event to event there can be different people who joins the event so keeping peers inside the event will be a good approach in my opinion.